mantle/rbx_mantle/src/config.rs

```diff
@@ -523,6 +523,20 @@ pub struct ExperienceTargetConfig {
 
     /// Spatial voice configuration.
     pub spatial_voice: Option<SpatialVoiceTargetConfig>,
+
+    /// Notification strings configuration.
+    pub notifications: Option<Vec<NotificationTargetConfig>>,
+}
+
+#[derive(JsonSchema, Serialize, Deserialize, Clone)]
+#[serde(rename_all = "camelCase", deny_unknown_fields)]
+pub struct NotificationTargetConfig {
+    /// The display name of the notification string on the Roblox website.
+    pub name: String,

```

Comment:
Since the name has to be unique, and since it's only visible to the creator, I wonder if we should just use the label from the HashMap as the name. We could make this an optional field if the creator wants to use a different name.